### ISSUE TITLE
page titles should be unique and describe purpose of the page

### ISSUE THREAD

**Issue by clhenrick** [clhenrick/clhenrick.io#55]
Some page titles aren't unique and default to "clhenrick.io":
- work/
- work/[project]/
- blog/tags/

Others have only the h1 text in their title:
- about/cv/
- about/talks/
- blog/
- blog/archive/
- blog/[post]/
- blog/tags/[tag]/
- contact/
- search/

Ideally each page title (other than the home page) should be in the form of "{{ unique name}} - Chris Henrick" so that it makes sense when read out of context. Titles should be concise and free of any special characters such as "|".

**Comment by clhenrick** [comment]
Fixed in #57
